x11-plugins/gkrellm-leds/gkrellm-leds-0.8.2-r2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gkrellm-plugin autotools
+
+MY_P="${P/rellm-/}"
+
+DESCRIPTION="GKrellM2 plugin for monitoring keyboard LEDs"
+HOMEPAGE="http://heim.ifi.uio.no/~oyvinha/gkleds/"
+SRC_URI="http://heim.ifi.uio.no/~oyvinha/e107_files/downloads/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~ppc ~sparc ~x86"
+
+RDEPEND="
+	app-admin/gkrellm:2[X]
+	x11-libs/libXtst"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+
+PATCHES=( "${FILESDIR}/${P}-r2-ldflags-typo.patch" )
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	PLUGIN_SO=( src/.libs/gkleds$(get_modname) )
```

Comment:
is this really needed?

Reply:
This should be the [name of the plugin's .so file](https://devmanual.gentoo.org/eclass-reference/gkrellm-plugin.eclass/index.html#lbAG). The default is `${PN}$(get_modname)`, so I think it's needed, yes.

Reply:
FWIW, referencing .libs/ tends to break with slibtool

Reply:
What's the solution to that? Move the file after it's been built?

Reply:
Do we actually need to set it at all in configure? Can't we just set it to the right name on install!